specification/attributes/column_naming_convention.md

```diff
@@ -19,16 +18,21 @@ Naming convention for columns appearing in billing data.
 
 ## Requirements
 
-* Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
-* Column IDs MUST NOT use abbreviations.
-* Column IDs SHOULD NOT use acronyms.
-* Column IDs MUST be alphanumeric with no special characters.
-* Columns that have an ID and a Name MUST have the Id or Name suffix in the Column ID. Display Name for a Column MAY
-  avoid the Name suffix if it is considered superfluous
+* All columns defined by FOCUS MUST follow the following rules:
+  * Column IDs MUST use [Pascal case](https://techterms.com/definition/pascalcase).
+  * Column IDs MUST NOT use abbreviations.
+  * Column IDs SHOULD NOT use acronyms.
+  * Column IDs MUST be alphanumeric with no special characters.
+  * Columns that have an ID and a Name MUST have the `Id` or `Name` suffix in the Column ID. Display Name for a Column MAY avoid the Name suffix if there are no other columns with the same name prefix.
+* Custom columns SHOULD follow the same rules as FOCUS columns listed above.
+* All custom columns MUST be prefixed with an underscore (`_`) to distinguish them from FOCUS columns and avoid conflicts in future releases.
```

Comment:
```suggestion
* All provider-defined columns MUST be prefixed with an underscore (`_`) to distinguish them from FOCUS columns and avoid conflicts in future releases.
```

Reply:
Columns starting with '_' might require special treatment by different DB/query engines hence cause unnecessary issues for practitioners. For example '_' needs to be encapsulated in Amazon Athena and it's one of the commonly used services for querying cost and usage data on AWS (https://docs.aws.amazon.com/athena/latest/ug/tables-databases-columns-names.html#names-that-begin-with-an-underscore). My opinion that using pattern 'provider_customname' would help to avoid this. 
I understand that you won't be able to group all custom columns by sorting, but at the same time all modern IDEs or query editors have search filters which allow quickly search for columns from particular provider by name